package.json

```diff
@@ -11,6 +11,8 @@
         "build:prod": "node --max_old_space_size=16384 ./node_modules/webpack/bin/webpack.js --mode production",
         "build:dev": "node --max-old-space-size=16384 ./node_modules/webpack/bin/webpack.js --mode development",
         "clean": "shx rm -rf public/bundle* mdx/tools",
+        "test": "jest --passWithNoTests",
```

Comment:
could you add this to the `pre_push` command as well?

Reply:
yup!

Reply:
added